### ISSUE TITLE
Can not find the Data source and Service selection

### ISSUE THREAD

**Issue by novemberrain11** [SAP-samples/fiori-elements-opensap#7]
Hello Stefan,

Thanks for your answer on another issue. I could not find the selection as you have shown in your guidance.
After selecting: ABAP Environment on SAP Business Technology Platform, I do not have the option to choose **default_abap-trial**.

Could you please help? Thanks.

![image](https://user-images.githubusercontent.com/87578356/127623378-5b2ed5af-548d-484a-9fdb-b848fc69ab5e.png)

**Comment by novemberrain11** [comment]
Hello again,  I got another error when generate the query. Could you please help to check? Thanks.

![image](https://user-images.githubusercontent.com/87578356/127637612-caca6048-5778-4247-8044-09f1540bd9ef.png)

**Comment by stefan0816** [comment]
Hey,

in the exercises we use SAP Business Application Studio, you are using Visual Studio Code. There are a few tiny differences. You can find more information here: https://help.sap.com/viewer/17d50220bcd848aa854c9c182d65b699/Latest/en-US/99061814ead548808d539861fb27bafb.html. 

Regarding the second screenshot, at which exercise and step do you get this error message?

**Comment by novemberrain11** [comment]
Hello Stefan,
Sorry for my late response. I have tried to use SAP BAS but I could not go further, that's why I switch to Visual Studio Code:
![image](https://user-images.githubusercontent.com/87578356/128606471-3bf32b5d-4a07-4c63-9716-dc6a74294d79.png)

Regarding the second screenshot, I have tried to replicate the issue, but now, I only got this error in the terminal, the Internet explorer is not even pop up.
Would it help you with the below screenshot?

![image](https://user-images.githubusercontent.com/87578356/128606574-97034895-2f9e-42ac-b239-1dbf915c4450.png)

Thanks for your support, I am pretty new to VS code and Fiori, and it is pretty hard to follow the OpenSAP + Github as the data sources is not in sync (in the Github and in openSAP course, you are using different data sources). Thank you.

**Comment by stefan0816** [comment]
Hey,

regarding the error message in your first screenshot, did you log on to Cloud Foundry? This is described here: https://github.com/SAP-samples/fiori-elements-opensap/blob/9e634e9446302a698cbfc8da364b6d69af8266d3/week2/unit4.md#step-1-ensure-you-are-logged-in-to-cloud-foundry. Could you perform this step and then try again? 

Regarding the second error, it's hard to tell without knowing which exercise you did and at which step you are currently. In the majority of the cases the participants accidentally missed a step of the exercise which then led to the issue. I also see you named the project "project1" and not like suggested in the exercise. It could be helpful if you start over with the current app you are building and carefully check, that you do all the described steps. For most participants that was the quickest solution.

**Comment by stefan0816** [comment]
Closing due to inactivity.
